CHANGELOG.md

```diff
@@ -37,6 +37,8 @@
 
 ### Added
 
+* nostr: add MLS tags and event kinds ([erskingardner])
```

Comment:
Can you add also the link to your github profile or website at the bottom of the file, in the "Contributors" section?

```md
<!-- Contributors -->
...
[erskingardner]: <link>
```